operations/patch.go

```diff
@@ -160,7 +156,18 @@ func PatchFile() cli.Command {
 				return errors.Wrap(err, "problem accessing evergreen service")
 			}
 
-			if _, err = params.validatePatchCommand(ctx, conf, ac, comm); err != nil {
+			ref, err := params.validatePatchCommand(ctx, conf, ac, comm)
+			if err != nil {
+				return err
+			}
+			var featureBranch string
+			if params.Ref == "" {
+				featureBranch = "HEAD"
```

Comment:
with a patch-file, is it not feasible that after generating the patch the user has checked out something else or even `cd`ed out of the repo?

Reply:
It's certainly conceivable. @john-m-liu @tychoish Any idea how people actually use this feature? I'd prefer to require that the user have the commit checked out, but I'm not sure what the actual use case is for patch-file.

Reply:
Actually, thinking about this again, I don't think this works. For `evergreen patch`, Evergreen does the diff, so we can do it against head. For a patch-file, we don't know what its base is.